entries/a/agl.com.au.json

```
@@ -0,0 +1,15 @@
+{
+  "AGL Energy": {
+    "domain": "agl.com.au",
+    "tfa": [
+      "sms"
```

According to the docs, both `sms` and `email` are supported.
```suggestion
      "sms",
      "email"
```